spring-cloud-gcp-pubsub-stream-binder/src/main/java/com/google/cloud/spring/stream/binder/pubsub/properties/PubSubConsumerProperties.java

```diff
@@ -32,6 +32,8 @@
 
 	private Integer maxFetchSize = 1;
 
+	private String customSubscription = null;
```

Comment:
I think I would name it `customSubscriptionName` or just `subscriptionName`.

Reply:
Done.